examples/open-loop/plot_loop.py

```diff
@@ -1,19 +1,16 @@
-import sys
-
-sys.path.append('../')
-
 import numpy as np
-from utilities.parser import Parser
+from eyeloop.utilities.parser import Parser
 import matplotlib.pyplot as plt
 import matplotlib.gridspec as gridspec
 import matplotlib.ticker as mticker
 
 
 class Loop_parser(Parser):
-    def __init__(self: Parser):
+    def __init__(self):
+        super().__init__(animal='human')
         self.set_style()
 
-    def custom_lower_panel_ticks(self: Parser, y, pos) -> str:
+    def custom_lower_panel_ticks(self, y, pos) -> str:
```

Comment:
While defining the `self` type is generally harmless, here are calling methods of the `Loop_parser`'s `self` instance, that are not available in `self: Parser` (parent type). Easier to leave it undefined, which by default assumes that's the same type of the current class (which is a `Parser` too)